Dockerfile

```diff
@@ -15,6 +18,9 @@ RUN pub get
 ADD app /project/app
 RUN pub get --offline
 
+# Flutter setup runs only if GAE_SERVICE=="analyzer"
+RUN /project/app/script/setup-flutter.sh
```

Comment:
It's a nice idea, but I'm not sure if `GAE_SERVICE` is available during docker build time. Did you try this out?

I think the environment variables mentioned [here](https://cloud.google.com/appengine/docs/flexible/python/upgrading) are only guaranteed to be available during execution of the container. Even if it happens to work, it's quite fishy and I would not rely on it.

As mentioned on the original PR, we can just sacrifice the 400 MB for now to get going.